### PR TITLE
Adds a New Drink, the Flaming Homer

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

Adds a new drink, the flaming homer. 

source: 
image

## Why It's Good For The Game

More drinks are fun. 

## Changelog
:cl:
add: A new drink, the Flaming Homer
/:cl:

unabashed reference code, which I normally dislike, almost didn't make the PR bc of it. but I think its fun and doesn't really ruin immersion at all, if you want I could change the Homer name to the name of whoever made it or something.

### CODE REVIEW

**Comment by Detective-Google** [comment]
more drink more better 👍

**Comment by Krysonism** [comment]
They have this on paradise but called "flaming moe".

I like this PR though.

**Comment by wesoda25** [comment]
oh i probably wont be able to fix that

**Comment by wesoda25** [comment]
> They have this on paradise but called "flaming moe".
> 
> I like this PR though.

he stoel the dirnK!!

**Comment by wesoda25** [comment]
Dammit i was using an older version of the drink sprites ill fix that later

**Comment by fludd12** [comment]
You could make it a combination VA-11HALL-A and Simpsons reference with the Flaming Moai drink, if you wanted.

**Comment by optimumtact** [comment]
We're reaching the point where more drinks == better is not actually true

**Comment by wesoda25** [comment]
> We're reaching the point where more drinks == better is not actually true

I disagree. I've been playing chef and bartender pretty often later and new content is always welcome. I believe its a different issue than chemical bloat.

**Comment by OverlyGeneric** [comment]
> You could make it a combination VA-11HALL-A and Simpsons reference with the Flaming Moai drink, if you wanted.

I am moyai and I approve of this message.

**Comment by Krysonism** [comment]
> 
> 
> We're reaching the point where more drinks == better is not actually true

Disagree, even if a drink is identical to another it will still improve bar gameplay by offering a new sprite and taste description. 

Variety is very important for the bar or kitchen gameplay.

**Comment by optimumtact** [comment]
\>I disagree

good for you

**Comment by wesoda25** [comment]
Sorry oranges I figured you might value the opinion of people who play the roles these pieces of code interact with.

**Comment by dr-bee** [comment]
Does this actually do anything? because there already is a flaming drink and its the toxins special. Which isnt a reference to something that has no relationship to space or sci-fi.

**Comment by Dennok** [comment]
Make it name from name of creator like "flaming [creator.name]"

**Comment by wesoda25** [comment]
> Make it name from name of creator like "flaming [creator.name]"

Ya I was considering that if people had a big issue with the reference

> Does this actually do anything? because there already is a flaming drink and its the toxins special. Which isnt a reference to something that has no relationship to space or sci-fi.

ya it makes you drunk bc its an alcoholic beverage

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
you reverted a few drink sprites

**Comment by ExcessiveUseOfCobblestone** [comment]
If drinks should be added they should have a unique aspect to them besides the name and sprite imo.

**Comment by wesoda25** [comment]
I mean if maintainers don’t like it I’ll just close the PR. As I said, I was conflicted on PRing it in the first place so it won’t really tear me up.

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -781,3 +781,11 @@
 	id = /datum/reagent/consumable/red_queen
 	results = list(/datum/reagent/consumable/red_queen = 10)
 	required_reagents = list(/datum/reagent/consumable/tea = 6, /datum/reagent/mercury = 2, /datum/reagent/consumable/blackpepper = 1, /datum/reagent/growthserum = 1)
+
+/datum/chemical_reaction/flaming_homer
+	name = "Flaming Homer"
+	id = /datum/reagent/consumable/ethanol/flaming_homer
+	results = list(/datum/reagent/consumable/ethanol/flaming_homer = 11)
+	required_reagents = list(/datum/reagent/consumable/ethanol/tequila = 5, /datum/reagent/consumable/ethanol/creme_de_menthe = 5, /datum/reagent/medicine/spaceacillin = 1) //spaceacillan standing in for cough syrup aka codeine
```
Comment (81Denton) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]: Imo replace spaceacilin with morphine since both codeine and morphine are opiates

**Comment by plapatin** [comment]
why cant we have more chef recipes instead of more drink recipes, chef's barely gotten any love lately

**Comment by wesoda25** [comment]
> why cant we have more chef recipes instead of more drink recipes, chef's barely gotten any love lately

i might add some new donut types [tm]

**Pull request closed without merging**